packages/rn-tester/js/examples/Appearance/AppearanceExample.js

```diff
@@ -89,7 +90,8 @@ exports.examples = [
         const colorScheme = useColorScheme();
         return (
           <RNTesterThemeContext.Provider
-            value={colorScheme === 'dark' ? themes.dark : themes.light}>
+            value={colorScheme === 'dark' ? themes.dark : themes.light}
```

Comment:
:') I was wondering how we were forcing PlatformColors to be dark in light mode. Turnsout we were hardcoding the hex strings :'(